crates/runtime/tests/models/snapshots/integration_models__models__s3_vectors__search__filters_pushdown_explain.snap

```diff
@@ -10,23 +10,21 @@ snapshot_kind: text
 |               |   Sort: vector_search(delivery, Utf8("wrong location")).package_weight_kg DESC NULLS FIRST, vector_search(delivery, Utf8("wrong location")).score DESC NULLS FIRST, fetch=10                                                                                                                                                                                                                                    |
 |               |     Projection: vector_search(delivery, Utf8("wrong location")).message.body, vector_search(delivery, Utf8("wrong location")).attempt_count, vector_search(delivery, Utf8("wrong location")).message.status, vector_search(delivery, Utf8("wrong location")).package_weight_kg, round(vector_search(delivery, Utf8("wrong location")).score, Int64(1)), vector_search(delivery, Utf8("wrong location")).score   |
 |               |       BytesProcessedNode                                                                                                                                                                                                                                                                                                                                                                                        |
-|               |         TableScan: vector_search(delivery, Utf8("wrong location")) projection=[message.body, message.status, attempt_count, package_weight_kg, score], full_filters=[vector_search(delivery, Utf8("wrong location")).attempt_count > Int64(1), vector_search(delivery, Utf8("wrong location")).package_weight_kg > Float64(5), vector_search(delivery, Utf8("wrong location")).message.status = Utf8("FAILED")] |
```

Comment:
The primary difference in these snapshots should be
1. The ordering off projection column names (now alphabetical)
2. In the logical plan, no filter above the table scan (it'll be a full_filter property of the table scan).